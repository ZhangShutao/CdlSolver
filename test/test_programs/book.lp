book(R) :- business(R), not reserved(R), not book(R1), R != R1, ticket(R1).
book(R) :- economy(R), not reserved(R), not book(R1), R != R1, ticket(R1), not &c{not book(R)}.
ticket(R) :- business(R).
ticket(R) :- economy(R).

business(b11).
economy(a11).
economy(a12).

reserved(b11).

goal :- 1{book(R)}1.
:- not goal.

#show book/1.
#show -book/1.
