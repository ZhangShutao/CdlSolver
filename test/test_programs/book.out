default model: 1
x:book(a11); y:book(a11);
Satisfiable